recipes/nspyre/meta.yaml

```diff
@@ -0,0 +1,94 @@
+{% set name = "nspyre" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}-org/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256:
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - nspyre = nspyre.gui:main
+    - nspyre-config = nspyre.config:main
+    - nspyre-mongodb = nspyre.mongodb:main
+    - nspyre-inserv = nspyre.inserv:main
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True
```

Comment:
This means that it is skipping all builds :cry:

Reply:
@scopatz these were some pretty big oversights on my part 😞I clearly did not have my head together this morning, but let's try this again